sapi_demo/src/EventSubscriber/RequestViewEventSubscriber.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Drupal\sapi_demo\EventSubscriber;
+
+use Drupal\sapi\Dispatcher;
+use Drupal\sapi\StatisticsItem;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Event\GetResponseForControllerResultEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+/**
+ * Class RequestViewEventSubscriber
+ */
+class RequestViewEventSubscriber implements EventSubscriberInterface {
```

Comment:
any reason to no implement Drupal\Core\DependencyInjection\ContainerInjectionInterface by adding create() ?